maxim/logger/gemini/utils.py

```
@@ -148,36 +156,39 @@ def parse_content(
                 if content.role == "model":
-                    message.role = "system"
+                    message["role"] = "system"
                 else:
-                    message.role = content.role
+                    message["role"] = content.role
             if content.parts is not None:
                 for part in content.parts:
                     if part is not None and part.text is not None:
-                        message.content += part.text
+                        message["content"] += part.text
         elif isinstance(content, List):
```

Ruff (0.11.9)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)